client/my-sites/site-settings/form-general.jsx

```diff
@@ -129,15 +129,16 @@ export class SiteSettingsFormGeneral extends Component {
 
 		return (
 			<FormFieldset>
-				<FormLabel htmlFor="wpversion">{ translate( 'WordPress Version' ) }</FormLabel>
-				<FormInput
-					name="wpversion"
-					id="wpversion"
-					data-tip-target="site-title-input"
-					type="text"
-					value={ get( selectedSite, 'options.software_version' ) }
-					disabled
-				/>
+				<FormLabel 
+					className="site-settings__wordpress-version" 
+					htmlFor="wpversion"
```

Comment:
Can this be changed to a different component? ie, not `FormLabel` – there is no input with ID `wpversion` for this to label anymore (and if somehow there is, in some other component on the page, it shouldn't attach to this label)

Reply:
👍 

Updated the PR to just use a `<strong>` tag, would that be okay?

Reply:
Better now! :-) There's also `FormFieldset` component which isn't semantically correct now.

Reply:
I updated it to use Fragment. :)